ui/app/templates/components/http-requests-table.hbs

```diff
@@ -11,15 +11,17 @@
       </tr>
     </thead>
     <tbody>
-      {{#each countersWithChange as |c|}}
+      {{#each (reverse countersWithChange) as |c|}}
         <tr>
-          <th scope="row" class="has-text-black has-text-weight-semibold">{{ date-format c.start_time 'MMMM YYYY' }}</th>
-          <td>{{c.total}}</td>
+          <th scope="row" class="has-text-black">{{ date-format c.start_time 'MMMM YYYY' }}</th>
```

Comment:
My chart and table both say "May" even though the counter starts at `2019-06-01T00:00:00Z` and I should only have stats for July. Is it possible the months are starting at 0 and need a `+ 1` for parsing?

I'm seeing the same thing in Storybook but didn't in the screenshot you posted in chat. Maybe a recent change?

Reply:
you're right -- this is because the date on the chart & table was being displayed in the users local timezone, whereas the counters are recorded in UTC. I think I fixed this by creating a new date with only the month and year, but I could use a sanity check.

Reply:
fixed!